guides/getting-started.md

```diff
@@ -153,8 +153,21 @@ If you want to use the `mat-icon` component with the official
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 ```
 
-For more information on using Material Icons, check out the
-[Material Icons Guide](https://google.github.io/material-design-icons/).
+Additionally you have to import `MatIconModule` to your project.
+
+```ts
+import {MatButtonModule, MatCheckboxModule, MatIconModule} from '@angular/material';
```

Comment:
You should probably have just `MatIconModule`.

Reply:
I'm not sure, I was trying to keep it consistent with previous examples in the same file.

Reply:
@julianobrasil It's okay. I'm fine with it.